src/main/kotlin/com/sourcegraph/cody/history/ChatHistoryPanel.kt

```diff
@@ -41,7 +41,7 @@ import javax.swing.tree.DefaultTreeModel
 import javax.swing.tree.TreePath
 import javax.swing.tree.TreeSelectionModel
 
-class HistoryTree(
+class ChatHistoryPanel(
```

Comment:
NTOE: I piggyback this change.

This class is a panel actually, it extends `SimpleToolWindowPanel`.